code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -33,6 +34,13 @@
 		M.hallucination += 5
 	..()
 
+/datum/reagent/drug/highjack/overdose_process(mob/living/M)
+	var/obj/item/bodypart/head/head = M.get_bodypart("head")
+	if(prob(30))
+		to_chat(M, "<span class='userdanger'>You feel really lightheaded...</span>")
+		head.dismember()
+
+
```

Comment:
Please move these into a NSV file as well (and remove that added first line in the file.